### PR TITLE
copr: vectorize degrees

### PR DESCRIPTION
**Pull request by AerysNan**
Signed-off-by: AerysNan <aerysnan@gmail.com>

PCP #5751

###  What have you changed?

Vectorize degrees.

###  What is the type of the changes?

Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?

Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No.

###  Does this PR affect `tidb-ansible`?

No.

### CODE REVIEW

File: components/tidb_query/src/rpn_expr/impl_math.rs [hunk 1]
```diff
@@ -253,6 +253,12 @@ fn sqrt(arg: &Option<Real>) -> Result<Option<Real>> {
     }))
 }
 
+#[inline]
+#[rpn_fn]
+fn degrees(arg: &Option<Real>) -> Result<Option<Real>> {
+    Ok(arg.and_then(|n| Some(Real::from(n.to_degrees()))))
```
Comment (breeswish) [inline components/tidb_query/src/rpn_expr/impl_math.rs]: Please use `Real::new()` to avoid panic for NaN cases.

**Comment by breeswish** [comment]
Please also create a PR in https://github.com/tikv/copr-test to enable testing your function. Thanks!

**Comment by AerysNan** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/44

**Comment by AerysNan** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/44

**Comment by AerysNan** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/44

**Comment by AerysNan** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/44

**Comment by breeswish** [comment]
@sticnarf @lonng @andylokandy @iosmanthus

**Comment by lonng** [comment]
copr-test PR: https://github.com/tikv/copr-test/pull/44

**Comment by AerysNan** [comment]
@breeswish Seems that auto-merge wasn't triggered?

**Comment by AerysNan** [comment]
@sre-bot /run-all-tests

**Comment by lonng** [comment]
@AerysNan Please resolve the conflicts, thanks.

**Comment by AerysNan** [comment]
@lonng done. Kindly re-review it at your convenience.

**Pull request merged**